src/test/lambda_log_shipper/export_logs_handlers/test_s3_handler.py

```diff
@@ -39,6 +39,21 @@ def test_format_records(record):
     assert data.decode() == expected
 
 
+def test_format_records_with_prefix_disabled(record, monkeypatch):
+    monkeypatch.setattr(Configuration, "disable_log_prefix", "true")
```

Comment:
```suggestion
    monkeypatch.setattr(Configuration, "disable_log_prefix", True)
```
This field is a boolean